main.py

```diff
@@ -35,19 +37,19 @@
 # The following is setup code that is required to host with modal.com
 image = Image.debian_slim().pip_install_from_requirements("requirements.txt")
 # Rename "poe-bot-quickstart" to your preferred app name.
```

Comment:
This needs to be updated too. Though actually I think the old name is fine here so I'd just revert the next line.